filament_scale_enhanced/__init__.py

```diff
@@ -35,25 +35,63 @@ def get_settings_defaults():
             spool_weight=200,
             clockpin=21,
             datapin=20,
-            lastknownweight=0
+            lastknownweight=0,
         )
 
     @staticmethod
     def get_assets():
         return dict(
             js=["js/filament_scale.js"],
             css=["css/filament_scale.css"],
-            less=["less/filament_scale.less"]
+            less=["less/filament_scale.less"],
         )
 
+    def __init__(self):
+        self.mqtt_publish = lambda *args, **kwargs: None
+        self.mqtt = False
```

Comment:
Should we be calling super? It's been a little while since I've looked at the classes and can't recall if this overrides anything.

Reply:
I don't think it overrides anything. 

Its directly from foosel's example here:
https://github.com/OctoPrint/OctoPrint-MQTT/tree/master#example

It's also a practice used in the documentation sample project.  I have seen it in a number of other PlugIns as well. Does not make it right but it does seem to be accepted.

Reply:
Just refreshed myself, the plugin classes are mixins. So the only init would whatever is on this class, so no issue here!